shell/gpu/gpu_surface_gl.cc

```diff
@@ -144,7 +157,16 @@ bool GPUSurfaceGL::CreateOrUpdateSurfaces(const SkISize& size) {
     return false;
   }
 
+  if (delegate_->UseOffscreenSurface()) {
+    offscreen_surface = CreateOffscreenSurface(context_.get(), size);
+    if (offscreen_surface == nullptr) {
+      FXL_LOG(ERROR) << "Could not create offscreen surface.";
+      return false;
+    }
+  }
```

Comment:
Is it worth nulling `offscreen_surface_` up above for consistency with our behaviour re the onscreen surface (line 141)?

Reply:
Yup, good catch.